images/virtio-container-disk/Dockerfile

```diff
@@ -0,0 +1,34 @@
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2017 Red Hat, Inc.
+#
+
+FROM fedora:28 as builder
```

Comment:
could we use something more stable? e.g. centos. Or anything small/minimal/stable

Reply:
Every image we use for upstream is based on fedora. Therefore it make sense to base even this one on Fedora.

Reply:
there are 2 F27 already in the code base, F28 will slowly get outdated too. I see in 2 cases "FROM fedora" - would that work? There's nothing specific to a Fedora version here

We should use something better than a random Fedora from the time the code was written originally. But that's a common problem for all the containers in kubevirt, so indeed doesn't have to be solved here

Reply:
Would you kindly please point to specific code locations where do you see it is broken? Thanks in advance.